src/components/timeline/Selection.js

```diff
@@ -249,7 +249,8 @@ class TimelineRulerAndSelection extends React.PureComponent<Props> {
       event.pageX < rect.left ||
       event.pageX >= rect.right ||
       event.pageY < rect.top ||
-      event.pageY >= rect.bottom
+      event.pageY >= rect.bottom ||
+      width === 0
```

Comment:
Do you want a comment in the code or is a comment here good enough?

The reason was that when moving the mouse around over the profiler page while the profiler was still loading, this code was triggered, with width == 0, causing the mouseTimePosition to become Infinity, and my new code to explode attempting to number format NaN.

I think a cleaner fix would be finding a way to register the mousemove handler later when the profile has finished loading, but returning early when width === 0 seemed good enough to avoid the crash.